src/processor/processor.module.ts

```diff
@@ -43,14 +46,24 @@ import { ProviderWebhookService } from './provider-webhook.service';
     BullModule.registerQueue({
       name: 'graphUpdateQueue',
       defaultJobOptions: {
-        attempts: 1,
+        attempts: 3,
         backoff: {
           type: 'exponential',
         },
         removeOnComplete: true,
-        removeOnFail: true,
+        removeOnFail: false,
       },
     }),
+
+    // Bullboard UI
+    BullBoardModule.forRoot({
```

Comment:
If we're adding BullBoard, might be able to get rid of some of the other endpoints we built?

Reply:
Not really you can just do retries mostly from bullboard apart from its visual esthetics